iphone/Classes/PlatformModule.m

```diff
@@ -220,28 +221,38 @@ - (NSString *)identifierForVendor
 }
 
 #if defined(USE_TI_PLATFORMIDENTIFIERFORADVERTISING) || defined(USE_TI_PLATFORMGETIDENTIFIERFORADVERTISING)
-- (NSNumber *)isAdvertisingTrackingEnabled
+- (BOOL)isAdvertisingTrackingEnabled
```

Comment:
This is a general question: Is our proxy able to return numeric values like `BOOL`, `int`, `NSInteger` or `CGFloat` as a Number? I noticed some crashes with this before, which required them to be `NSNumber` boxed.

Reply:
I believe so? https://developer.apple.com/documentation/javascriptcore/jsvalue?language=objc

CGFloat is a typedef for `double`. Looks like `NSNumber` and primitive numeric types are handled.